File: rollingDice.cc
#include <bits/stdc++.h>
using namespace std;

int n,f,r;
int fav = 0, cases = 1;

void b(int pos, int val) {
    if (pos == n) {
        if (val == r) ++fav;
        return;
    }
    for (int i = 1; i <= f; ++i) b(pos+1, val+i);
}

int main() {
    cin >> n >> f >> r;
    
    for (int i = 0; i < n; ++i) cases *= f;
    
    if (n*f >= r) {
        b(0,0);
    }
    
    cout << "The probability of getting a " << r <<  " with " << n << " dice of " << f << " faces is " << setprecision(3) << fixed << double(fav)/double(cases) << endl;
}

File: rollingDice.py
from random import randint as rng
MOBE = input().split()
n = int(MOBE[0])
c = int(MOBE[1])
v = int(MOBE[2])
a = 0
for i in range(10000):
	s = 0
	for j in range(n):
		s += rng(1, c)
	if s == v:
		a+=1
print("The probability of getting a "+str(v)+" with "+str(n)+" dice of "+str(c)+" faces is " +str(round(a/100000, 3)))

